src/table/builder.rs

```diff
@@ -91,9 +100,9 @@ impl Builder {
             return;
         }
         for offset in &self.entry_offsets {
-            self.buf.put_u32_le(*offset);
+            self.buf.put_u32(*offset);
```

Comment:
Why change this?

Reply:
I'll change this back.

Reply:
And I'll use little endian for all data in SST, to have a unified interface.

Reply:
Now data in SST are all stored using little endian.

Reply:
seems badger use big-endian? Though I don't know if there is any difference between big-endian and little-endian for encode/decode.

Reply:
Badger now uses flatbuffer, and it seems to be little endian now...